.github/workflows/GNU.yml

```diff
@@ -83,7 +104,7 @@ jobs:
         GNULIB_DIR="${PWD}/gnulib"
```

Comment:
This and the above variable is not used. Same with `SKIP` and `XPASS` variables below.